### PR TITLE
chore: remove broken endpoints

### PR DESCRIPTION
**Pull request by ojeytonwilliams**
These still exist, technically, but there's no render engine so they all
generate errors instead of pages

Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

### CODE REVIEW

**Review by huyenltnguyen — APPROVED** [review]
LGTM

File: api-server/src/server/boot/randomAPIs.js [hunk 1]
```diff
@@ -14,36 +14,10 @@ module.exports = function (app) {
   router.get('/u/:email', unsubscribeDeprecated);
   router.get('/unsubscribe/:email', unsubscribeDeprecated);
   router.get('/ue/:unsubscribeId', unsubscribeById);
-  router.get(
-    '/the-fastest-web-page-on-the-internet',
```
Comment (nhcarrigan) [inline api-server/src/server/boot/randomAPIs.js:18]: image

Is this handled elsewhere then?

Comment (ojeytonwilliams) [reply api-server/src/server/boot/randomAPIs.js:18]: It is, yes. It's handled by Gatsby these days: https://github.com/freeCodeCamp/freeCodeCamp/tree/main/client/static/the-fastest-web-page-on-the-internet

**Review by naomi-lgbt — APPROVED** [review]
Should be good to go then

**Pull request merged**